ubid.rb

```diff
@@ -68,6 +68,7 @@ class UBID
   TYPE_FIREWALL_RULE = "fr"
   TYPE_FIREWALL = "fw"
   TYPE_POSTGRES_FIREWALL_RULE = "pf"
+  TYPE_USAGE_ALERT = "a1"
```

Comment:
Seems unfortunate to sit on what is presumably an abbreviation of "alert" (a1ert) for _usage_ records only. Seems hardly the most important or dominant kind of alert, if we want to build an alerting channel, which we probably will at some point. Though I suppose you could try to pivot the meaning (e.g. make UsageAlert a join on an Alert table and backfill) to rectify it post-facto.

Reply:
I changed it to va as abbreviation of "vsage alert"